scripts/fregion.py

```diff
@@ -9,11 +9,19 @@
 #
 #    to read the stored field 'x' out of f:
 #      f.x
-#
+       
+#    If x is an array type, it is lazy loaded, and it supports the interface as following:
+#      1) for d in f.x()
```

Comment:
For this case would it be possible to make it 'for d in f.x'?  It's probably just a question of supporting that 'getitems' function or whatever it is right?

Reply:
yes, you can . you can see the class RecStream, it is just a wrapper of f.x

Reply:
In that case maybe the comment should be updated to say that you can write it as 'for d in f.x()'?  Just a little simpler from the API perspective.